apps/perf-test/tasks/perf-test.ts

```diff
@@ -341,10 +360,12 @@ function createScenarioTable(scenarioSettings, testResults, showAll) {
     resultsToDisplay
       .map(key => {
         const testResult = testResults[key];
-        const { iterations } = scenarioSettings[key] || {};
+        const { scenarioName, iterations, renderType } = scenarioSettings[key] || {};
+        const renderTypeFriendlyName = renderType === 'VirtualRerender' ? 'Virtual re-render' : renderType;
```

Comment:
nit: I think the casing and `re-render` dashing is awkward here too. I think it's ok to just reflect `mount` and `virtual-rerender` in the report. Casing implies a unique object rather than a type, so I think all lower casing is actually more clear for the render type.